docs/detections/rules-ui-create.asciidoc

```diff
@@ -355,6 +355,10 @@ also affect this rule.
 .. *Building block* (optional): Select to create a building-block rule. By
 default, alerts generated from a building-block rule are not displayed in the
 UI. See <<building-block-rule>> for more information.
+.. *Indicator prefix override*: Specify the default threat indicator path that the indicator match rule will use when obtaining threat indicator data from the defined indicator indices. This data is used to enrich indicator match alerts with metadata about matched threat indicators. The default value for this field is `threat.indicator`. 
++
+IMPORTANT: If your threat indicator data is at a different location, update this setting with the appropriate field prefix to ensure alert enrichment can still be performed.
```

Comment:
```suggestion
IMPORTANT: If your threat indicator data is at a different location, update this setting with the appropriate field prefix to enable alert enrichment.
```

Reply:
I like the succinctness of your recommendation, but am concerned it might give the impression that this setting can be turned on and off, and we really don't ever want users to turn it off or leave the setting undefined. I believe the main idea we're trying to convey here is that, if users have non-ECS-compliant indices, their threat fields will be located at a different path. So, to ensure they're still able to view threat intel on their indicator match alerts, they should define that custom path here.  @ecezalp feel free to correct me if I'm wrong.

Reply:
+1 to `ensure alert enrichment can still be performed` because threat_indicator_path is a mandatory field